drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-api/src/main/java/org/drools/workbench/screens/scenariosimulation/service/AssetsService.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.scenariosimulation.service;
+
+import java.util.List;
+
+import org.jboss.errai.bus.server.annotations.Remote;
+
+/**
+ * Assets specific service
+ */
+@Remote
+public interface AssetsService {
+
+    /**
+     * This method uses only <code>String</code> to avoid importing specific <b>backend</b> implementations where they should not be needed nor available
+     * (e.g. avoid using uberfire vfs stuff inside kogito environment)
+     * @param containerDirectoryFullPath
+     * @param assetType
+     * @param packageName
+     * @return
+     * @throws Exception
+     */
+    List<String> getAssets(final String containerDirectoryFullPath,
```

Comment:
Longer term I think Kogito should really wrap these Strings with something meaningful. It has already been bad enough mess to figure out what path is project root, module root or just a path to a different branch. But this can not be fixed here.
Could be that this will not end up being that big of a deal if there is no knowledge of different branches or modules.

Reply:
@Rikkola 
k